nautobot_version_control/templates/nautobot_version_control/diff_detail.html

```diff
@@ -4,6 +4,7 @@
 {% load helpers %}
 {% load static %}
 {% load tz %}
+<!-- This view comes from clicking on the "Diff Type" icon, e.g. changed/added in a diff view -->
```

Comment:
I could not update this to the standard templates, the view is rather different so I am not sure if this is expected or not.